amd_openvx_extensions/amd_rpp/source/internal_publishKernels.cpp

```diff
@@ -44,94 +44,95 @@ vx_status get_kernels_to_publish()
+    STATUS_ERROR_CHECK(ADD_KERNEL(MeanStddev_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(FlipbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ResizebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ResizeCropbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(RotatebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(WarpAffinebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(FisheyebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(LensCorrectionbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ScalebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(WarpPerspectivebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(DilatebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ErodebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(HuebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(SaturationbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ColorTemperaturebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(VignettebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ChannelExtractbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ChannelCombinebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(LookUpTablebatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(BoxFilterbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(SobelbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(MedianFilterbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(CustomConvolutionbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(NonMaxSupressionbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(GaussianFilterbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(NonLinearFilterbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(LocalBinaryPatternbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(DataObjectCopybatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(GaussianImagePyramidbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(LaplacianImagePyramid_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(CannyEdgeDetector_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(HarrisCornerDetector_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(FastCornerDetector_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(remap_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(TensorAdd_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(TensorSubtract_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(TensorMultiply_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(TensorMatrixMultiply_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(TensorLookup_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ColorTwistbatchPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(CropMirrorNormalizePD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(CropPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ResizeCropMirrorPD_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(Copy_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(Nop_Register));
+    STATUS_ERROR_CHECK(ADD_KERNEL(ResizeMirrorNormalizeTensor_Register));
```

Comment:
Can we add the ResizeMirrorNormalizeTensor_Register at the end, as it shows it as a big change. This is just a one liner change

Reply:
These are changes requested by Lakshmi for the RMN PR